hamocc/mo_param_bgc.F90

```diff
@@ -513,6 +522,29 @@ module mo_param_bgc
   real :: beta13, alpha14, d14cat, d13c_atm
 
 contains
+  !********************************************************************
+  subroutine ini_bgctimes(calendar)
+
+    character(len=*),intent(in) :: calendar
+
+    select case (trim(calendar))
+      ! see mod_calendar.F90 and mod_time.F90
+      case ('noleap','365_day')
+        days_per_year = 365.
+      case ('all_leap','366_day')
```

Comment:
```suggestion
      case ('all_leap','366_day')
      if (mnproc == 1) then
         write (io_stdo_bgc,*) 'WARNING: Init iHAMOCC time variables: non-standard 366_day calendar selected'
      endif      
```

Reply:
I'll do so. Sorry, I overlooked your previous comment on doing all this in a different PR (was already working on it...) - shall I try to split it up or is it ok to proceed?